codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/MemberModel.java

```diff
@@ -377,33 +377,48 @@ private boolean returnTypeIs(Class<?> clazz) {
     }
 
     public String getFluentSetterDocumentation() {
-        StringBuilder docBuilder = new StringBuilder();
-        docBuilder.append(getSetterDocumentation())
-                .append(LF)
-                .append("@return " + stripHtmlTags(DEFAULT_FLUENT_RETURN))
-                .append(getEnumDoc());
-        return docBuilder.toString();
+        return getSetterDocumentation()
+               + LF
```

Comment:
what does LF abbr stand for (saw the member declaration)?

Reply:
line-feed